src/bootstrap/src/core/build_steps/llvm.rs

```diff
@@ -153,10 +154,9 @@ pub fn prebuilt_llvm_config(builder: &Builder<'_>, target: TargetSelection) -> L
 /// This retrieves the LLVM sha we *want* to use, according to git history.
 pub(crate) fn detect_llvm_sha(config: &Config, is_git: bool) -> String {
     let llvm_sha = if is_git {
-        helpers::get_closest_merge_base_commit(
+        get_closest_merge_commit(
```

Comment:
This here makes it look like the logic for finding the LLVM commit will change. So I am confused by your statement saying that it will not change.

Reply:
We just moved bootstrap function to build_helper crate.